designer/client/src/components/graph/PanZoomPlugin.ts

```diff
@@ -84,10 +84,11 @@ export class PanZoomPlugin {
         this._enabled = enabled;
     }
 
-    fitContent(content: g.Rect, updatedViewport?: Viewport): void {
+    fitContent(content: g.Rect, updatedViewport?: Viewport, predefinedScale?: number | undefined): void {
         this.viewport = updatedViewport;
 
-        const scale = Math.min(1, 0.78 / Math.max(content.width / this.viewport.width, content.height / this.viewport.height));
+        const autoScaleValue = Math.min(1, 0.78 / Math.max(content.width / this.viewport.width, content.height / this.viewport.height));
```

Comment:
also maybe name the 0.78 something?

Reply:
I'm not sure what is the proper name for this number, I just named it `zoomThreshold`